server/monitor.go

```diff
@@ -2162,7 +2163,6 @@ type AccountInfo struct {
 	Claim       *jwt.AccountClaims   `json:"decoded_jwt,omitempty"`
 	Vr          []ExtVrIssues        `json:"validation_result_jwt,omitempty"`
 	RevokedUser map[string]time.Time `json:"revoked_user,omitempty"`
-	RevokedAct  map[string]time.Time `json:"revoked_activations,omitempty"`
```

Comment:
As a 2.7.1, I don't really like that we "break" this by removing an exported field, but since it was a bug, I guess that is fine. But some applications will fail to compile when updating their dependencies to v2.7.1...

Reply:
This is part of an account debug call.
I poked at the jsm library and it appears to not be used there.

Should I leave the entry in there and comment it as obsolete?

Reply:
No, that would not really help. The application would be "broken" anyway without a change to go get the information from the exports. Thanks for checking.